server/jetstream_api.go

```diff
@@ -121,8 +121,11 @@ const (
 	// JSDirectMsgGet is the template for non-api layer direct requests for a message by its stream sequence number or last by subject.
 	// Will return the message similar to how a consumer receives the message, no JSON processing.
 	// If the message can not be found we will use a status header of 404. If the stream does not exist the client will get a no-responders or timeout.
-	JSDirectMsgGet  = "$JS.DS.GET.*"
```

Comment:
Were those already used in a public release or used by client libraries that have been released?

Reply:
No..